livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -689,6 +689,41 @@ def __init__(self, sess: RealtimeSession) -> None:
         def create(self) -> None:
             self._sess._queue_msg({"type": "response.create"})
 
+        async def acreate(self, *, cancel_existing: bool = True) -> None:
+            # FIXME: should we merge this with `create`?
+            pending_create_fut = self._sess._response_create_fut
+            if pending_create_fut is not None:
+                if not cancel_existing:
+                    return
+                await pending_create_fut
+
+            pending_resp_id = self._sess._active_response_id
+            if pending_resp_id:
+                if not cancel_existing:
+                    logger.warning(
```

Comment:
I think there are three behaviors right?
1. canceling existing response, enqueuing new response
2. not canceling existing response, aborting the new response
3. not canceling existing response, enqueuing new response

it seems 1 or 2 are the most common path, but for the sake of API design, should it be:
* on_duplicate: "cancel_existing" | "cancel_new" | "keep_both" ?

Reply:
not loving the name `acreate` either.. `create` seems so good.. but.. maybe `async request`, like `await response.request()` ?

just an idea. .not sure if I love it either

Reply:
Yes, there are three options. I'll update the API. I think `create` is good but it's originally a sync method, not sure if it's ok to change it to an async method which might be a breaking changes as I saw many users are calling `response.create` in the entrypoint.

Reply:
right, we do not want to break the existing API for that reason.

Reply:
Modified the `create` to return a `Future` or a `Task`, how do you think?